### ISSUE TITLE
my trainning data is thchs30, however it met some questions, and I  modify something, could I test the model in the end?Now,it starts training...

### ISSUE THREAD

**Issue by Michael1S** [Rayhane-mamah/Tacotron-2#246]
1.trainning data is thchs30,  and modify "./preprocess.py" line 49:
```
return [os.path.join('/home/cext/xiaoyi', 'data_thchs30')]
```
----
I have a question when `python preprocess.py`, should I have change the language? The default language is en_US, I did nothing in the language.

2.trainning data is thchs30, and update "./datasets/preprocessor.py"

in 33 lines:
```
trn_files = glob.glob(os.path.join(input_dir, 'xmly_record', 'A*', '*.trn'))
```
then, print(trn_files), however, it's empty list, so I made this update:
```
# trn_files = glob.glob(os.path.join(input_dir, 'xmly_record', 'A*', '*.trn'))
# ===update 20181018====
trn_files = glob.glob(os.path.join(input_dir, 'train', 'A*.trn'))
```

the last question, after training finished, could I test the model in Chinese? Oh, anyway, that is a good project.

**Comment by gloriouskilka** [comment]
Hello, @Michael1S

1) 
> I have a question when python preprocess.py, should I have change the language? The default language is en_US, I did nothing in the language.


Have you read main README file? How many times? https://github.com/Rayhane-mamah/Tacotron-2/blob/master/README.md

> Step (0): Get your dataset, here I have set the examples of Ljspeech, en_US and en_UK (from M-AILABS).

> Note:
> 
> Our preprocessing only supports Ljspeech and Ljspeech-like datasets (M-AILABS speech data)! If running on datasets stored differently, you will probably need to make your own preprocessing script.

When I was preprocessing my non-English data, I just looked at the code, found place where the data is preprocessed, found where all these parameters like language are used, and made my own modifications. Your question doesn't seem to be related to this project, it's like python-related question.

2) 
> the last question, after training finished, could I test the model in Chinese? Oh, anyway, that is a good project.

"Synthesis" paragraph in README

**Comment by gloriouskilka** [comment]
PS I'm not maintainer of this project, I'm just a user

**Comment by Michael1S** [comment]
hahahhh, lao ge 666
U a right. I am ok with u.

**Comment by Michael1S** [comment]
Have u read this file ?
[preprocess.py](https://github.com/begeekmyfriend/Tacotron-2/blob/mandarin-new/preprocess.py)
for 39 lines and 47 lines, this project had contained how to prepross the dataset-Thchs30.
```
# 39 line
supported_datasets = ['LJSpeech-1.0', 'LJSpeech-1.1', 'M-AILABS', 'THCHS-30']
# 47 line
if args.dataset.startswith('THCHS-30'):
    return [os.path.join(args.base_dir, 'data_thchs30')]
```

**Comment by Michael1S** [comment]
My question is not preprocess.py, is when running "python preprocess.py", the file "./dataset/preprocessing.py" returns a empty list. So I modified something in  the file "./dataset/preprocessing.py". I am not sure wether it is right or the model could be test in Chinese.The author had not reminded of this change, I tried it myself.

**Comment by gloriouskilka** [comment]
> Have u read this file ?
> preprocess.py
> for 39 lines and 47 lines, this project had contained how to prepross the dataset-Thchs30.

No, I haven't. You gave a link to somebody's fork of this repo. You can ask him if you have any questions about his modifications containing in his `mandarin-new` branch of his fork.
